### ISSUE TITLE
[MozFest] Footer links 

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4824]
### Background:
We will change the footer label "Media Center", using the term "Press Center" instead so it is consistent to our other pages.

### Tasks:

- [ ] In the https://www.mozillafestival.org/ footer, replace the label "Media Center" by "Press Center" 

- [ ] Reorder the items by alphabetical order (that works on Desktop, Tablet and Mobile views), except "Donate" label, which should be the first item due to its importance.

### Link:
The footer link will take people to this page: https://www.mozillafestival.org/en/media/

### Desired final state:
<img width="1315" alt="image" src="https://user-images.githubusercontent.com/22157921/86019484-847d4500-b9db-11ea-979f-18892871b405.png">

@TheoChevalier tagging you here as this may have implications for localized pages. Kevin mentioned in another ticket that this page should be kept in English so people know that who will read it is an English speaker. However we may consider localize the label "Press Center" in all languages we have the footer on.

**Comment by taisdesouzalessa** [comment]
UPDATE: we will add copy to the license notice on the bottom of the page - ticket #4868 so I wonder if it makes more sense to wait on this other ticket so when the engineer modifies this footer they can do all the changes at once? Let me know if you prefer this approach @cadecairos or if you think separated tickets makes more sense.

**Comment by cadecairos** [comment]
I don't have a problem combining the footer copy changes. But let's close one of the issues and merge it into the other, along with the details

**Comment by taisdesouzalessa** [comment]
So let's combine them, it seems more efficient. I'll add a note and close all the footer tickets for now, opening them once we have the other footer changes ready to go. Thanks Cade!

**Comment by taisdesouzalessa** [comment]
**Note:** let's wait on ticket #4868 to be resolved then open a new ticket with all footer changes at once.
